script.js

```diff
@@ -0,0 +1,462 @@
+                        document.getElementsByClassName(`${event.which}`)[1].style.backgroundColor = '';
+                    break;
+
+                    case 'Shift' :
+                        document.getElementsByClassName(`${event.which}`)[0].style.backgroundColor = '';
+                        document.getElementsByClassName(`${event.which}`)[1].style.backgroundColor = '';
+                    break;
+
+                        default:
+            document.getElementsByClassName(`${event.which}`)[0].style.backgroundColor = '';
+            document.getElementsByClassName(`${event.which}`)[0].style.borderRadius = '';
+            
+            
+            
+            
+            }
+        });
+
+
+        buttons.forEach(button =>  {
+            const buttonElement = document.createElement('li');
+            buttonElement.classList.add(`${elementWhich[i]}`)
+            const lineBreak = ["backspace", "\\", "enter", "rshift"].indexOf(button) !== -1;
+            buttonElement.classList.add('usual');
+            let buttonImg = document.createElement('span');
+            buttonImg.classList.add('eng');
+            buttonImg.classList.add(`${buttonsCode[i]}`);
+            buttonImg.id = `${buttons[i]}`;
+            let rusButtonImg = document.createElement('span');
+            rusButtonImg.id = `${rusButtons[i]}`;
+            rusButtonImg.classList.add('ru');
+            rusButtonImg.classList.add(`${buttonsCode[i]}`);
+            let shiftButton = document.createElement('span');
+            shiftButton.id = `${shiftButtons[i]}`;
+            shiftButton.classList.add('shiftBut');
+            shiftButton.classList.add('hidden');
+            shiftButton.classList.add(`${buttonsCode[i]}`);
+            let shiftButtonPic = document.createElement('span');
+            shiftButtonPic.classList.add('ShiftButPic');
+            let rusShiftButtonPic = document.createElement('span');
+            rusShiftButtonPic.classList.add('rusShiftButPic');
+            let rusShiftButton = document.createElement('span');
+            rusShiftButton.classList.add('rusShiftBut');
+            rusShiftButton.classList.add('hidden');
+            rusShiftButton.classList.add(`${buttonsCode[i]}`)
+            this.properties.ru == 'true' ? buttonImg.classList.add('hidden') : rusButtonImg.classList.add('hidden');
+            this.properties.ru == 'true' ? shiftButtonPic.classList.add('hidden') : rusShiftButtonPic.classList.add('hidden');
+            buttonImg.insertAdjacentText('afterbegin', buttons[i]);
+            rusButtonImg.insertAdjacentText('afterbegin', rusButtons[i]);
+            shiftButton.insertAdjacentText('afterbegin', shiftButtons[i]);
+            rusShiftButton.insertAdjacentText('afterbegin', rusShiftButtons[i]);
+            shiftButtonPic.insertAdjacentText('afterbegin', shiftButtonsPic[i]);
+            rusShiftButtonPic.insertAdjacentText('afterbegin', rusShiftButtonsPic[i]);
+            i++;
```

Comment:
Now we met this value (i) and don't have any clue why and for what reason it's here)